app/(service)/my-store/edit/_components/myStoreForm.tsx

```diff
@@ -0,0 +1,168 @@
+'use client';
+
+import { useState } from 'react';
+import { Controller, FieldValues, useForm } from 'react-hook-form';
+
+import Image from 'next/image';
+
+import Button from '@/components/button/button';
+import Input from '@/components/input/input';
+import SelectInput from '@/components/input/selectInput';
+
+// 3. SelectInput의 옵션 메뉴 position absolute 주기
+const FOOD_CATEGORY_LIST = [
+  { id: 1, category: '한식' },
+  { id: 2, category: '중식' },
+  { id: 3, category: '일식' },
+  { id: 4, category: '양식' },
+  { id: 5, category: '분식' },
+  { id: 6, category: '카페' },
+  { id: 7, category: '편의점' },
+  { id: 8, category: '기타' },
+];
+
+const ADDRESS_LIST = [
+  { id: 1, category: '서울시 종로구' },
+  { id: 2, category: '서울시 중구' },
+  { id: 3, category: '서울시 용산구' },
+  { id: 4, category: '서울시 성동구' },
+  { id: 5, category: '서울시 광진구' },
+  { id: 6, category: '서울시 동대문구' },
+  { id: 7, category: '서울시 중랑구' },
+  { id: 8, category: '서울시 성북구' },
+  { id: 9, category: '서울시 강북구' },
+  { id: 10, category: '서울시 도봉구' },
+  { id: 11, category: '서울시 노원구' },
+  { id: 12, category: '서울시 은평구' },
+  { id: 13, category: '서울시 서대문구' },
+  { id: 14, category: '서울시 마포구' },
+  { id: 15, category: '서울시 양천구' },
+  { id: 16, category: '서울시 강서구' },
+  { id: 17, category: '서울시 구로구' },
+  { id: 18, category: '서울시 금천구' },
+  { id: 19, category: '서울시 영등포구' },
+  { id: 20, category: '서울시 동작구' },
+  { id: 21, category: '서울시 관악구' },
+  { id: 22, category: '서울시 서초구' },
+  { id: 23, category: '서울시 강남구' },
+  { id: 24, category: '서울시 송파구' },
+  { id: 25, category: '서울시 강동구' },
+];
+
+export default function MyStoreForm() {
+  const {
+    register,
+    handleSubmit,
+    control,
+    formState: { errors },
+  } = useForm<FieldValues>({ mode: 'all' });
+  const [imageSrc, setImageSrc] = useState<any>(null);
+
+  const handleUploadImage = (e: React.ChangeEvent<HTMLInputElement>) => {
+    if (e.target.files && e.target.files[0]) {
+      const file = e.target.files[0];
+      const reader = new FileReader();
+      reader.readAsDataURL(file);
+      reader.onload = () => {
+        setImageSrc(reader.result);
+      };
+    }
```

Comment:
P5: 옵셔널 체이닝을 사용하면 조금 더 간결하게 파일 존재 여부를 확인해 볼 수 있을 것 같습니다.

```suggestion
    const file = e.target.files?.[0];
    if (file) {
      const reader = new FileReader();
      reader.readAsDataURL(file);
      reader.onload = () => setImageSrc(reader.result);
    }
```